cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -306,11 +306,11 @@ private void copyStream(ReadOnlyDataSource from, DataSource to, String fromName,
             "The triplestore used during the import",
             TripleStoreFactory.defaultImplementation())
             .addAdditionalNames("powsyblTripleStore");
-    private static final Parameter PROFILE_USED_FOR_INITIAL_STATE_VALUES_PARAMETER = new Parameter(
-            PROFILE_USED_FOR_INITIAL_STATE_VALUES,
-            ParameterType.STRING,
-            "Profile used for initial state values",
-            "SSH");
+    private static final Parameter READ_SHUNT_SECTIONS_TAP_POSITIONS_FROM_SV_IF_PRESENT_PARAMETER = new Parameter(
```

Comment:
Can you keep the previous name as additional name for compatibility?

Reply:
Done. I made additional modifications to keep the type of the parameter to allow the previous parameter for compatibility.